types/blueimp-load-image/blueimp-load-image-tests.ts

```diff
@@ -22,11 +22,22 @@ const b64DataJPEG =
   '2uLj5OXm5+jp6vLz9PX29/j5+v/aAAwDAQACEQMRAD8A/v4ooooA/9k=';
 const imageUrlJPEG = 'data:image/jpeg;base64,' + b64DataJPEG;
 
+// Callback style
 loadImage(imageUrlJPEG, (image: Event | HTMLCanvasElement | HTMLImageElement, data?: loadImage.MetaData): void => {
   const canvas = image as HTMLCanvasElement;
   console.log(data);
   canvas.toBlob((blob: Blob | null): void => {
     const url = canvas.toDataURL("image/png");
     console.log(url);
   });
-}, {canvas: true, orientation: true, maxWidth: 100, maxHeight: 100, crop: true});
+}, { canvas: true, orientation: true, maxWidth: 100, maxHeight: 100, crop: true });
+
+// Promise style
+loadImage(imageUrlJPEG, { canvas: true, orientation: true, maxWidth: 100, maxHeight: 100, crop: true }).then((data) => {
+  const canvas = (<any> data.image) as HTMLCanvasElement;
```

Comment:
Why is the cast to `any` needed?

Reply:
The existing typings have `false` as a possibility for `image` (if we don't use canvas I'm guessing)

![image](https://user-images.githubusercontent.com/9100419/100948094-02c7af80-3557-11eb-932c-dcaa04a3d7db.png)

Which means you can't directly type-cast a potential `boolean` into a `HTMLCanvasElement` without first casting to `unknown` or `any`

![image](https://user-images.githubusercontent.com/9100419/100948171-2f7bc700-3557-11eb-8a6d-fa28138d8315.png)